crates/oxc_linter/src/rules/jest/no_alias_methods.rs

```diff
@@ -106,43 +106,39 @@ impl Rule for NoAliasMethods {
         jest_node: &PossibleJestNode<'a, 'c>,
         ctx: &'c LintContext<'a>,
     ) {
-        run(jest_node, ctx);
-    }
-}
+        let node = jest_node.node;
+        let AstKind::CallExpression(call_expr) = node.kind() else {
+            return;
+        };
+        let Some(jest_fn_call) = parse_expect_jest_fn_call(call_expr, jest_node, ctx) else {
+            return;
+        };
+        let Some(matcher) = jest_fn_call.matcher() else {
+            return;
+        };
+        let Some(alias) = matcher.name() else {
+            return;
+        };
```

Comment:
The changes to this file are unrelated refactoring and don't change anything to do with `MemberExpression`, right?

(if that's correct, just for future reference, it would have been preferable to do this in a separate PR - the diff of this PR is hardly minimal already!)

Reply:
Woops, sorry about that! I was deep into debugging the jest fn parsing logic and I had rearranged/refacotred things and reverted all of it, but apparently I forgot to revert this.